lib/opFns.js

```diff
@@ -43,15 +44,15 @@ module.exports = {
     } else {
       r = a.div(b).toArray('be', 32)
     }
-    return new Buffer(r)
+    return Buffer.from(r)
   },
   SDIV: function (a, b, runState) {
     a = utils.fromSigned(a)
     b = utils.fromSigned(b)
 
     var r
     if (b.isZero()) {
-      r = new Buffer([0])
+      r = Buffer.alloc(1)
```

Comment:
Thinking a bit more about this it may make the code more readable if not using `alloc(1)`, but rather `Buffer.from([0])` in `opFns.js`.